mobile_config_cli/src/cmds/mod.rs

```diff
@@ -0,0 +1,216 @@
+use crate::{cmds::env::NetworkArg, KeyRole, NetworkKeyRole, Result};
+use anyhow::Context;
+use clap::{Args, Parser, Subcommand};
+use helium_crypto::PublicKey;
+use std::path::PathBuf;
+
+pub mod admin;
+pub mod authorization;
+pub mod entity;
+pub mod env;
+pub mod gateway;
+
+pub const ENV_CONFIG_HOST: &str = "HELIUM_CONFIG_HOST";
+pub const ENV_CONFIG_PUBKEY: &str = "HELIUM_CONFIG_PUBKEY";
+pub const ENV_KEYPAIR_BIN: &str = "HELIUM_KEYPAIR_BIN";
+
+#[derive(Debug, Parser)]
+#[command(name = "mobile-config")]
+#[command(author, version, about, long_about=None)]
+pub struct Cli {
+    #[command(subcommand)]
+    pub command: Commands,
+
+    #[arg(
+        global = true,
+        long,
+        env = ENV_CONFIG_HOST,
+        default_value = "https://config.mobile.mainnet.helium.io:6080"
+    )]
+    pub config_host: String,
+
+    #[arg(
+        global = true,
+        long,
+        env = ENV_CONFIG_PUBKEY,
+        default_value = "137oJzq1qZpSbzHawaysTGGsRCYTXG1MiTMQNxYSsQJp4YMDdN8"
```

Comment:
If someone has access to the server’s signing key (@bbalser) it’d be nice to pull this value from it and update the PR with the correct value